.travis/script.sh

```diff
@@ -67,3 +67,17 @@ start_section "symbiflow.redir.2" "Running ${GREEN}make${NC} inside ${PURPLE}vpr
 end_section "symbiflow.redir.2"
 
 $SPACER
+
+start_section "symbiflow.configure_cmake" "Configuring CMake"
+mkdir build
+cd build
+cmake ..
+end_section "symbiflow.configure_cmake"
+
+$SPACER
+
+start_section "symbiflow.run_cmake_tests" "Running tests with CMake"
```

Comment:
I think these might be better as separate sections?

Reply:
Done